core/classifiers/CodeClassifier/CodeClassifier.py

```diff
@@ -0,0 +1,481 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Base class for classification algorithms"""
+
+from collections import Counter
+import keyword
+from StringIO import StringIO
+import token
+import tokenize
+
+from core.classifiers.BaseClassifier import BaseClassifier
```

Comment:
There is still one problem. `base` is top level directory name, inside which `BaseClassifier` module is present and inside this `BaseClassifier` class is implemented. So it'd be `from core.classifiers.base import BaseClassifier` and then use `BaseClassifier.BaseClassifier` as base class.

Should I create a file names `base` and move `BaseClassifier` to it?